app/src/main/java/com/youssef/fixit/Home/HomeActivity.java

```diff
@@ -40,7 +40,7 @@ public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
                         ReplaceFragment(new MoreFragment());
                         break;
                     case R.id.add_work:
-                        if(MainActivity.MyRole.equals("client")){
+                        if(SplashScreen.MyRole.equals("client")){
```

Comment:
If you have fixed/static strings it's better to keep them in one place and access those strings using the reference for example you have "client" here as fixed and used across the application so it will be better if you created a `Constants` class and add ROLE_CLIENT = "client" and use the string using ROLE_CLIENT instead. This way it will be easier to maintain and whenever you need to change this hardcoded strings will be easier to change it from one place.

Reply:
ignore it for now will address in the next pr